docs/content/how-to/aws/index.md

```diff
@@ -0,0 +1,957 @@
+    style AWSCL fill:#FEA175,stroke:#333,stroke-width:3px
+    style SVC fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style RT fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style CMHCP fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style PVC fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style SCS fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style HCP fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style APPS fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style CL fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style SC fill:#BAE0E9,stroke:#333,stroke-width:3px
+
+
+    HO ==> SVC
+    HO ==> RT
+    HO ==> CMHCP
+    HO ==> PVC
+    HO ==> SCS
+    HO ==> APPS
+    HO ==> HCP
+    HO ==> AWSCL
+    HO ==> CL
+    HO ==> SC
+
+    subgraph OMC[Openshift Management Cluster]
+    direction LR
+        subgraph HONS[Hypershift Operator Namespace]
+            HO[fa:fa-server Hypershift Operator]
+        end
+        subgraph CANS[Cluster Namespace]
+            subgraph SC[Secrets]
+                KBCFGSC[admin-kubeconfig]
+                KADPWD[kubeadmin-password]
+            end
+        end
+        subgraph HCPN[Hosted Control Plane Namespace]
+            direction TB
+            subgraph SVC[Services]
+                CIPSVC[Cluster IP Services]
+                LBSVC[LoadBalancer Services]
+            end
+
+            subgraph RT[Routes]
+                ignition-Server
+                ovn-sbdb
+                oauth
+                konnectivity-server
+            end
+
+            subgraph CMHCP[ConfigMaps]
+                Configurations
+                Certificates
+                ...
+            end
+
+            subgraph SCS[Secrets]
+                KBC[Kubeconfig]
+                CA[Certificates]
+                SCETC[...]
+            end
+
+            subgraph PVC[PVCs]
+                etcd
+                ovn
+            end
+
+            subgraph APPS[ControlPlane Apps]
+                STS[Stateful Sets]
+                DP[Deployments]
+                RS[Replica Sets]
+            end
+
+            HCP <-.-> CL <-.-> AWSCL
+            subgraph HCP[Hosted Control Plane]
+                DescHCP("Contains the OCP
+                HostedCluster details")
+            end
+            subgraph AWSCL[AWSCluster]
+                DescAWSC("Contains the AWS details like
+                VPC, Region, Subnet, etc...")
+            end
+            subgraph CL[Cluster]
+                DescCL("Perform the relationship between
+                AWSCluster and HostedControlPlane")
+            end
+        end
+    end
+```
+> Legend:
+>
+> - Orange: OCP objects directly related with AWS.
+> - Cyan: Provider agnostic OCP objects.
+
+At the same time, the Hypershift operator will expect to exists the mentioned Infra components in the defined provider (You could use the Hypershift CLI in order to create them automatically). As a first step it will assign an InfraID to all the objects created in the AWS side.
```

Comment:
I'm having a hard time to follow this diagram. I'd expect us to present and list the cloud infra categorised in the following 4 buckets:
Pre-required unmanaged guest cluster aws account infra: e.g vpc (hypershift CLI is just a convenient automation for it, nothing else).
Infra managed by hypershift in management aws account . e.g LB for kas.
Infra managed by hypershift in guest cluster aws account. e.g nodepool instances.
Infra managed by kubernetes in guest cluster aws account. e.g PVs, service type LB.

Reply:
For the Resource / controllers relationships we also have https://hypershift-docs.netlify.app/reference/controller-architecture/ already

Reply:
I've modified a bit the style and added a new section to show this 4 buckets you mention. PTAL and I'll make the corrections needed (if make sense) :)

Reply:
Also added a new plugin for MKDocs which enlarge images smoothly. Mainly for the Networking diagram gif